pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java

```diff
@@ -527,17 +527,18 @@ private static Object generateRandomValue(Schema.Type fieldType) {
   /**
    * Run equivalent Pinot and H2 query and compare the results.
    */
-  static void testQuery(String pinotQuery, String brokerUrl, org.apache.pinot.client.Connection pinotConnection,
-      String h2Query, Connection h2Connection)
+  static void testQuery(ClusterTest testCluster, String pinotQuery, String brokerUrl,
```

Comment:
Do we need this change? I don't see usage of `testCluster`. `postQuery()` is a static method